### PR TITLE
Add susanne to about us page

### PR DESCRIPTION
**Pull request by chiaberry**
https://github.com/cityofaustin/atd-data-tech/issues/11813

https://deploy-preview-62--atd-product.netlify.app/about

Added Susanne to the team! Welcome!!

### CODE REVIEW

File: pages/about/index.js [hunk 1]
```diff
@@ -139,6 +139,12 @@ const TEAM_MEMBERS = [
     pronouns: "She/Her",
     src: "assets/headshots/christina_tremel.png",
   },
+  {
+    name: "Susanne Gov",
+    title: "Application Analyst",
+    pronouns: "she/her",
```
Comment (roseeichelmann) [inline pages/about/index.js:145]: maybe change to "She/Her" to follow the pattern of the others

Comment (chiaberry) [reply pages/about/index.js]: thank you, i copied/pasted and didnt pay attention to the capitalization. fixed!

**Review by patrickm02L — APPROVED** [review]
🎉  Hooray!

File: public/assets/headshots/susanne_gov.jpg
Comment (mddilley) [inline public/assets/headshots/susanne_gov.jpg:1]: The image file size came up last time around, but this seems reasonable to me. We have a few others that are around this size.

**Review by roseeichelmann — APPROVED** [review]
yay for new team members 🥳

**Review by mddilley — APPROVED** [review]
Welcome, Susanne! 🎉 🚀 🙌

**Pull request merged**